framework/OpenMod.Core/Eventing/EventBus.cs

```diff
@@ -155,7 +155,12 @@ public virtual void Unsubscribe(IOpenModComponent component, Type eventType)
             
             eventTypes.AddRange(@event.GetType().GetInterfaces().Where(d => typeof(IEvent).IsAssignableFrom(d)));
 
-            foreach (var eventType in eventTypes)
+            Type[] omittedTypes = new[]
```

Comment:
This should be declared as static readonly so it doesnt have to be allocated on each event emit

Reply:
Resolved by 02b8ba8